ios/RNVirgilGroupSession.m

```diff
@@ -0,0 +1,147 @@
+//
+//  RNVirgilGroupSession.m
+//  RNVirgilCrypto
+//
+//  Created by vadim on 9/19/19.
+//  Copyright © 2019 Virgil Security, Inc. All rights reserved.
+//
+
+#import "RNVirgilGroupSession.h"
+
+@implementation RNVirgilGroupSession
+
++ (BOOL)requiresMainQueueSetup
+{
+    return NO;
+}
+
+- (instancetype)init
+{
+    self = [super init];
+    self.rng = [VSCFCtrDrbg new];
+    self.keyProvider = [VSCFKeyProvider new];
+    [self.rng setupDefaultsAndReturnError:nil];
+    [self.keyProvider setupDefaultsAndReturnError:nil];
+    return self;
+}
+
+- (VSCFGroupSession *) createSession:(NSArray<NSString *> *) epochMessagesBase64 error:(NSError **)err
+{
+    NSMutableArray<VSCFGroupSessionMessage*> *epochMessages = [NSMutableArray arrayWithCapacity:[epochMessagesBase64 count]];
```

Comment:
Is autoreleasepool required here to avoid memory leaks?

Reply:
Good catch. I'll wrap all of the exposed method bodies in autoreleasepool